pkg/controller/disruption/disruption_test.go

```diff
@@ -1511,22 +1511,20 @@ func TestStalePodDisruption(t *testing.T) {
 			if _, err := dc.coreClient.CoreV1().Pods(tc.pod.Namespace).Create(ctx, tc.pod, metav1.CreateOptions{}); err != nil {
 				t.Fatalf("Failed to create pod: %v", err)
 			}
+			dc.clock.Sleep(tc.timePassed)
 			if err := dc.informerFactory.Core().V1().Pods().Informer().GetIndexer().Add(tc.pod); err != nil {
 				t.Fatalf("Failed adding pod to indexer: %v", err)
 			}
-			dc.clock.Sleep(tc.timePassed)
```

Comment:
why the sleep order change?

Reply:
to ensure the time passed before the object is processed by the worker